### DISCUSSION TITLE
A problem after installing Catalina, when the system starts booting, a kernel panic occurs

### DISCUSSION THREAD

**Discussion by cobraSY** [community/community#73353] [Apps, API and Webhooks]
### Select Topic Area

General

### Body

Do you have any solution?

After a very long struggle 🥹, I tried to install Catalina on a Nitro 5 laptop with an AMD Ryzen processor. Upon reaching the installation interface, I wiped the disk and installed the system. After around 2 minutes, it rebooted, displayed numerous texts, showed the Apple logo, counted, but ended abruptly. Then it restarted and displayed texts, stopping at this point. 

laptop = acer Nitro 5 AN515-43-R9YJ

CPU = AMD Ryzen 53550

GPU = RX560X 

Memory = 16 G 

Storage = 256 PCIe NVMe SSD 

![PXL_20231103_181712309](https://github.com/community/community/assets/148143759/25d90b29-eceb-4c5d-8308-9c1002919931)
![IMG_20231103_174656](https://github.com/community/community/assets/148143759/af740290-9d6d-4093-b76a-c532182c7469)
![IMG_20231103_174543](https://github.com/community/community/assets/148143759/ee3f3f29-ec42-4bb4-b5bd-c8764e58902c)
![IMG_20231103_174641](https://github.com/community/community/assets/148143759/f4217c69-9010-42a8-b5c2-d72c50dc6ae5)

**Comment by ghostinhershell** [comment]
Hi @cobraSY,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this issue, like [the Apple Support Community](https://discussions.apple.com/welcome). Although it is possible that another GitHub user might have run into this same issue and can help, the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
